locales/it/messages.json

```diff
@@ -1930,6 +1930,9 @@
     "receiverButtonBindMessage": {
         "message": "Richiesta di collegamento inviata al controllore di volo."
     },
+    "receiverButtonPassphrase": {
```

Comment:
Please rmeove all translations for no english language, they made via crowdin by our ttanslators, just add the english translation.